Cannot find name 'Location' when compiling React Native Typescript project

When I run `tsc` in my Typescript React Native project, I get the following error:

```
node_modules/navigation/navigation.d.ts:182:45 - error TS2304: Cannot find name 'Location'.

182     getUrl(hrefElement: HTMLAnchorElement | Location): string;
                                                ~~~~~~~~

node_modules/navigation/navigation.d.ts:230:45 - error TS2304: Cannot find name 'Location'.

230     getUrl(hrefElement: HTMLAnchorElement | Location): string;
                                                ~~~~~~~~

node_modules/navigation/navigation.d.ts:277:45 - error TS2304: Cannot find name 'Location'.

277     getUrl(hrefElement: HTMLAnchorElement | Location): string;
                                                ~~~~~~~~
```

Here are my `tsconfig` options:

```
"compilerOptions": {
    "allowJs": true,
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true,
    "isolatedModules": true,
    "jsx": "react",
    "lib": ["es6", "es2017", "es2019"],
    "moduleResolution": "node",
    "noEmit": true,
    "strict": true,
    "target": "esnext"
  },
```

If I add `dom` as a lib here, the error goes away - but then I get more errors since it's not a DOM project

--- a/build/npm/navigation-react-native/navigation.react.native.d.ts
+++ b/build/npm/navigation-react-native/navigation.react.native.d.ts
@@ -2,6 +2,10 @@ import { Component, Context, ReactNode } from 'react';
 import { BackHandler, ImageRequireSource, ImageURISource, NativeSyntheticEvent, StyleProp, ViewStyle } from 'react-native';
 import { Crumb, State, StateContext } from 'navigation';
 
+declare global {
+    interface Location {}
+}
+
 /**
  * Defines the Navigation Stack Props contract
  */

